.golangci.yml

```diff
@@ -0,0 +1,24 @@
+issues:
+  max-per-linter: 0
+  max-same-issues: 0
+
+linters:
+  disable-all: true
+  enable:
+    - durationcheck
+    - errcheck
+    - exportloopref
+    - forcetypeassert
+    - gofmt
+    - gosimple
+    - ineffassign
+    - makezero
+    - misspell
+    - nilerr
```

Comment:
Do we usually skip `paralleltest` for providers?

Reply:
We can discuss this one out of band -- providers tend to implement [`resource.ParallelTest()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#ParallelTest), which all its doing under the hood is call `t.Parallel()` and [`resource.Test()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#Test). I believe these would all show as false positives.

Reply:
Cool, I'll leave it off the providers then 👍🏻